docs/getting-started.md

```diff
@@ -67,20 +67,93 @@ KONNECTD_DEBUG_TOKEN
 : Token to grant metrics access, empty default value
 
 KONNECTD_DEBUG_PPROF
-: Enable pprof debugging, defaults to `false`
+: Enable pprof debugging
```

Comment:
why not adding back the default value here https://github.com/owncloud/ocis-konnectd/blob/master/pkg/flagset/flagset.go#L108

for boolean flags it can be done by adding:

```golang
&cli.BoolFlag{
	Name:        "debug-pprof",
	Value: 	     false,
	Usage:       "Enable pprof debugging",
	EnvVars:     []string{"KONNECTD_DEBUG_PPROF"},
	Destination: &cfg.Debug.Pprof,
},
```